rllib/policy/eager_tf_policy_v2.py

```diff
@@ -537,7 +538,10 @@ def compute_log_likelihoods(
 
         seq_lens = tf.ones(len(obs_batch), dtype=tf.int32)
         input_batch = SampleBatch(
-            {SampleBatch.CUR_OBS: tf.convert_to_tensor(obs_batch)},
+            {
+                SampleBatch.CUR_OBS: tf.convert_to_tensor(obs_batch),
+                SampleBatch.ACTIONS: actions,
```

Comment:
action was missing. forward_train() requires actions to be an existing key here.

Reply:
how did this work before?

Reply:
The RLModule's forward_train() function enforces the key existence today. In the past, the explore flag I think used to make it such that the code-path that used action keys did not get called.

Reply:
I see 👍